scripts/scr_restart_variables/scr_restart_variables.gml

```diff
@@ -1,9 +1,5 @@
+// Restarts variables
```

Comment:
👍  for extra documentation. However, just repeating the name of the script is not informative, we need some extra info.  In this case maybe: ", ensuring loaded saves are properly initialized"?